pkg/util/core.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build !windows
+
+package util
+
+import (
+	"fmt"
+	"runtime/debug"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"golang.org/x/sys/unix"
+)
+
+// SetCoreLimit enables core dumps and sets the core dump size limit based on configuration
+func SetCoreLimit() error {
```

Comment:
This could be renamed to clarify that the function sets up core dumps (i.e. enables + sets core limit), as opposed to simply setting the core limit?

```suggestion
func SetupCoreDump() error {
```

WDYT?

Reply:
Done!